charts/datree-admission-webhook/values.yaml

```diff
@@ -0,0 +1,59 @@
+# Default values for datree-admission-webhook.
+
+# The name of the namespace all resources will be created in.
+namespace: datree
+
+# The number of Datree webhook-server replicas to deploy for the webhook.
+replicaCount: 2
+
+# Additional labels to add to all resources.
+customLabels: {}
+
+# Create ClusterRoles, ClusterRoleBindings, and ServiceAccount
+rbac:
+  serviceAccount:
+    # Create the ServiceAccount
+    create: true
+    # The ServiceAccount name
+    name: webhook-server-datree
+  clusterRole:
+    # Create the ClusterRole
+    create: true
+    # The ClusterRole name
+    name: webhook-server-datree
+
+datree:
+  # The token used to link the CLI to your dashboard.
+  token: <DATREE_TOKEN>
+  # The name of the policy to check, e.g: staging. (string, optional)
+  policy: ""
+  # Display 'How to Fix' link for failed rules in output. (boolean ,optional)
+  verbose: ""
+  # The format output of the policy check results: yaml, json, xml, simple, JUnit. (string ,optional)
+  output: ""
+  # Don’t send policy checks metadata to the backend. (boolean ,optional)
+  noRecord: ""
+
+# The Datree webhook-server image to use.
+image:
+  # Image repository
+  repository: datree/admission-webhook
+  # Image tag
+  tag: 0.1.20
+  # Image pull policy
+  pullPolicy: Always
+
+# Security context for the containers
+securityContext:
+  allowPrivilegeEscalation: false
+  readOnlyRootFilesystem: true
+  runAsNonRoot: true
+  runAsUser: 25000
+
+resources:
+  limits:
+    cpu: 1000m
+    memory: 512Mi
+  requests:
+    cpu: 100m
```

Comment:
Don't use any resource request or limit. The user should do it

Reply:
But we won't follow our own standards 🥺